### ISSUE TITLE
Create disk sizes plugin [17]

### ISSUE THREAD

**Issue by matthewrsj** [osuosl/iam#24]
This is a meta-issue for #30, #31, and #32
Needs the following methods:

```
class disk_plugin
    def register { ... }  # migrations or migration logic
    def collect { ... }   # look in redis for information, store in db
    def report { ... }    # pull from db, output as csv and other

    register()            # call this in the class
```

**Comment by alxngyn** [comment]
we have an example of it from Kennric/examples
https://github.com/osuosl/iam/blob/kennric/examples/plugins/testo.rb

**Comment by ElijahCaine** [comment]
@matthewrsj Can this be closed? I'm not sure if we completed the disk size plugin or we've just made progress.

**Comment by matthewrsj** [comment]
@ElijahCaine I was going to close it with the merge of #56. I would like it to stay here because it is (semi) blocking #48

**Comment by alxngyn** [comment]
+1 to closing.
This issue references diskSize and #48 is for VCPU

**Comment by matthewrsj** [comment]
Resolved by #56
